subsys/bluetooth/controller/Kconfig

```diff
@@ -4,6 +4,36 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+choice BT_LL_CHOICE
+	prompt "Bluetooth Link Layer Selection"
+	default BT_LL_SOFTDEVICE
+
+config BT_LL_SOFTDEVICE
+	bool "SoftDevice Link Layer"
+	select MPSL
+	select ZERO_LATENCY_IRQS
+	select BT_CTLR_LE_ENC_SUPPORT
+	select BT_CTLR_EXT_REJ_IND_SUPPORT
+	select BT_CTLR_PRIVACY_SUPPORT
+	select BT_CTLR_EXT_SCAN_FP_SUPPORT
+	select BT_CTLR_PHY_UPDATE_SUPPORT
+	select BT_CTLR_CHAN_SEL_2_SUPPORT
+	select BT_CTLR_CONN_RSSI_SUPPORT
+	select BT_CTLR_DATA_LEN_UPDATE_SUPPORT
+	select BT_CTLR_ADV_EXT_SUPPORT
+	select BT_CTLR_ADV_PERIODIC_SUPPORT
+	select BT_CTLR_PHY_2M_SUPPORT if HAS_HW_NRF_RADIO_BLE_2M
+	select BT_CTLR_PHY_CODED_SUPPORT if HAS_HW_NRF_RADIO_BLE_CODED
+	select BT_HAS_HCI_VS
+	# The SoftDevice Controller only supports nRF52 and nRF53, it does not support nRF51
+	# or nRF91 (nRF91 doesn't even have the physical HW needed for BLE).
+	depends on (SOC_SERIES_NRF52X || SOC_NRF5340_CPUNET)
+	help
+	  Use SoftDevice Link Layer implementation.
+
+endchoice
+
+if BT_LL_SOFTDEVICE
 menu "Nordic BLE controller"
```

Comment:
When you are touching this file, you could also change this menu to "SoftDevice Controller"